docs/UserGuide.md

```diff
@@ -352,7 +380,7 @@ are taking each of the subjects.
 ## Utility Features
 ### Clearing all entries : `clear`
 
-Clears all student entries from the system.
+Clears all student entries from the system, and resets the studentID count to S00001.
```

Comment:
Good addition! Clarifies the issue with studentId resetting that was raised by testers.